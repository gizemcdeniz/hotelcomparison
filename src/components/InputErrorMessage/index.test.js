import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "./index";

it("renders correctly and matches the snapshot", () => {
  const mockError = {
    email: {
      message: "Please provide your email",
      type: "string.empty",
    },
  };

  const tree = renderer
    .create(<ErrorMessage error={mockError.email} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
