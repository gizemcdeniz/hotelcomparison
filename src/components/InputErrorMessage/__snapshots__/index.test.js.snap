// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<div
  className="errorMessage"
  role="alert"
>
  <img
    alt="Error logo"
    src="patchExclamationFill.svg"
  />
  <p>
    Please provide your email
  </p>
</div>
`;
